packages/core/content-releases/server/src/controllers/release-action.ts

```diff
@@ -17,6 +20,62 @@ const releaseActionController = {
       data: releaseAction,
     };
   },
+  async findMany(ctx: Koa.Context) {
+    const releaseId: GetReleaseActions.Request['params']['releaseId'] = ctx.params.releaseId;
+
+    const allowedContentTypes = getAllowedContentTypes({ strapi, userAbility: ctx.state.userAbility });
+    
+    const morphSanitizedPopulate = await allowedContentTypes.reduce(async (promiseAccumulator, contentTypeUid) => {
+      const accumulator = await promiseAccumulator;
```

Comment:
btw! there is a reduceAsync utility in strapi/utils , it might be useful in a case like this

Reply:
ohhh nice! 🍾